MailCore/Utils/FeatureFlagsManageable.swift

```diff
@@ -0,0 +1,31 @@
+/*
+ Infomaniak Mail - iOS App
+ Copyright (C) 2022 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import Foundation
+import SwiftUI
+
+public protocol FeatureFlagsManageable {
+    typealias UserId = Int
+    typealias AppFeatureFlags = [UserId: [FeatureFlag]]
```

Comment:
A quickwin could be using a [SendableDictionary.swift](https://github.com/Infomaniak/ios-core/blob/master/Sources/InfomaniakCore/Asynchronous/SendableDictionary.swift) it would make the manager thread safe for free.

probably looks like this `SendableDictionary<[UserId: [FeatureFlag]]>`

Reply:
(I think making an actor will be more troublesome, but better to do if you can make it work)